modules/gitops-creating-a-new-client-in-dex.adoc

```diff
@@ -0,0 +1,31 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+= Enabling the Dex openshift OAuth Connector
+
+Dex uses the users and groups defined within OpenShift by querying the platform provided `OAuth` server.
+
+. The `openShiftOAuth` property is used to trigger the Operator to auto configure the built-in OpenShift `OAuth` server.
```

Comment:
Is this a procedure? I'm thinking yes since the steps are numbered. If it is, then I would include a `Procedure` heading like you did for `Configuring SSO for Argo CD using Keycloak.`

I would also make the steps action oriented for users. See my suggestions below.

Reply:
Use the `openShiftOAuth` property to trigger the Operator to auto configure the built-in OpenShift `OAuth` server. 

Also, should it be `OpenShiftOAuth`? Please confirm as it surprises me that the S is capitalized but not the O.